frontend/src/metabase/nav/components/SearchResults.jsx

```diff
@@ -84,7 +85,8 @@ export default _.compose(
     query: (_state, props) => ({
       q: props.searchText,
       limit: DEFAULT_SEARCH_LIMIT,
-      models: props.models,
+      ...props.searchFilters,
+      models: props.searchFilters.type,
```

Comment:
Maybe it would be better to write a function called `convertSearchParamsToRequest` or something? There's only one or two places where this conversion (i.e. adding `models` to the query payload)